index.html

```diff
@@ -130,24 +130,24 @@ <h3 class="profession">I'am a Software Developer</h3>
       <li class="flex7">Ruby</li>
     </ul>
   </div>
-  <div class="button7"><button id="button7">See Project</button></div>
-</section>
-<section class="card8">
+  <div class="button7"><button id="button7" type="button">See Project</button></div>
+</div>
+<div class="card8">
   <div class="p8-title"><h1 id="p8-title">About me</h1></div>
   <div class="p8-text"><p id="p8-text">Hello I'm a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hestiate to contact me.</p></div>
-  <div class="button8"><button id="button8">Get My Resume</button></div>
-</section>
-<section class="card9">
+  <div class="button8"><button id="button8" type="button">Get My Resume</button></div>
+</div>
+<div class="card9">
   <ul class="skills">
-    <li class="skill-title"><span class="symbol">&#9674</span>Lenguages
+    <li class="skill-title"><span class="symbol">&#9674;</span>Lenguages
```

Comment:
- [ ] Kindly use the correct image and font style for your technology categories (Languages, Frameworks, and Skills) so as to follow the template 👍 
- Your design:
![Annotation 2022-03-18 060620](https://user-images.githubusercontent.com/85113567/158941857-370a24d6-9005-473f-a8c6-2202b476af82.jpg)
- Figma design:
![Annotation 2022-03-18 060610](https://user-images.githubusercontent.com/85113567/158941869-9f4cb1bb-db78-4dec-8703-8cbbb1e22203.jpg)